content/tutorials/portenta-h7/vs-openmv-ttn/content.md

```diff
@@ -180,5 +180,5 @@ The most common issue is that the device cannot connect to a TTN gateway. Again,
 If we are within good range of a gateway, we should also try to move our device and antenna to a window, and even hold it out the window and move it around. This has proven successful on numerous accounts, as the signal can travel less obstructed.
 
 **Authors:** Lenard George, Ignacio Herrera, Benjamin Dannegård 
-**Reviewed by:**
-**Last revision:**
\ No newline at end of file
+**Reviewed by:** Lenard George [2021-03-31]
```

Comment:
Needs two trailing spaces otherwise the line break doesn't work. https://gist.github.com/shaunlebron/746476e6e7a4d698b373

Reply:
Ok, was wondering why it was giving an error. Fixed it, thanks!